### PR TITLE
Feature/main feature

### PR DESCRIPTION
**Pull request by pacyL2K19**
# The aim of this PR 
- Creation of js files to manipulate the DOM 
- Add images to the project 
- Add multiple page and links between them

### CODE REVIEW

**Review by Forison — CHANGES_REQUESTED** [review]
## Invalid Request❕❓

#### Please not that, you have to make sure all linter checks are passing before you submit a review request, this P/R is therefore marked invalid🤗.

#### Kindly address the failing checks and submit a re-review.

#### Happy coding

Addo Forison

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- <link rel="shortcut icon" href="./images/favicon.ico" /> -->
```
Comment (Forison) [inline dist/index.html]: Always remove commented code.

**Review by Forison — CHANGES_REQUESTED** [review]
# Status::Change required☑️ 

#### Good job on addressing the failing checks. Your pull request, commit messages and readme are are professional. Your code is split into module as required The page looks great.
Here is one thing to do to get this approved.
* [ ]  It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

### Kindly address the issue and submit a re-review💪.

#### Happy coding👨‍💻

Addo Forison

File: dist/index.html (hunk 1 above)
Comment (pacyL2K19) [reply dist/index.html]: Thank you, done

**Review by NtwaliHeritier — APPROVED** [review]
### **STATUS: APPROVED✔✔✔**

Hello @pacyL2K19 
Great work making the requested changes.

You can merge👏👏

Happy coding💻💻💻

**Comment by pacyL2K19** [comment]
Thank you for the review @NtwaliHeritier

**Pull request merged**